docs/platforms/javascript/common/configuration/integrations/custom.mdx

```diff
@@ -8,4 +8,40 @@ In addition to the integrations that come with the SDK, you can also write custo
 
 Custom integration must conform to the [Integration interface](https://github.com/getsentry/sentry-javascript/blob/master/packages/types/src/integration.ts).
 
-<Include name="platforms/configuration/integrations/custom.mdx" />
```

Comment:
@krystofwoldrich Just a heads up that we also need to change this for RN when we bump to v8.